infra-as-code/bicep/modules/policy/assignments/alzDefaults/alzDefaultPolicyAssignments.bicep

```diff
@@ -98,6 +98,9 @@ param parDdosProtectionPlanId string = ''
 @description('Resource ID of the Resource Group for Private DNS Zones. Empty to skip assigning the Deploy-Private-DNS-Zones policy.')
 param parPrivateDnsResourceGroupId string = ''
 
+@description('Location of Private DNS Zones.')
+param parPrivateDnsZonesLocation string = 'eastus'
```

Comment:
Should this default to eastus? Or should it be a required input, for the same as the resource group or deployment location?

Reply:
Fixed in latest commit